src/PowerShellEditorServices/Services/PowerShell/Execution/SynchronousPowerShellTask.cs

```diff
@@ -34,19 +32,23 @@ internal class SynchronousPowerShellTask<TResult> : SynchronousTask<IReadOnlyLis
             _logger = logger;
             _psesHost = psesHost;
             _psCommand = command;
-            _executionOptions = executionOptions;
+            PowerShellExecutionOptions = executionOptions;
         }
 
+        public PowerShellExecutionOptions PowerShellExecutionOptions { get; }
+
+        public override ExecutionOptions ExecutionOptions => PowerShellExecutionOptions;
+
         public override IReadOnlyList<TResult> Run(CancellationToken cancellationToken)
         {
             _pwsh = _psesHost.CurrentPowerShell;
 
-            if (_executionOptions.WriteInputToHost)
+            if (PowerShellExecutionOptions.WriteInputToHost)
             {
                 _psesHost.UI.WriteLine(_psCommand.GetInvocationText());
             }
 
-            return !_executionOptions.NoDebuggerExecution && _pwsh.Runspace.Debugger.InBreakpoint
```

Comment:
I have no idea why I had this configuration before and I couldn't find any reference to it, so I got rid of it